src/System.Console/src/System/ConsolePal.Windows.cs

```diff
@@ -808,9 +793,9 @@ public static void SetCursorPosition(int left, int top)
                 // Give a nice error message for out of range sizes
                 int errorCode = Marshal.GetLastWin32Error();
                 Interop.Kernel32.CONSOLE_SCREEN_BUFFER_INFO csbi = GetBufferInfo();
-                if (left < 0 || left >= csbi.dwSize.X)
```

Comment:
Dead code: we checked for `left < 0` here: https://github.com/hughbe/corefx/blob/fbca63abe848380b8d07ebfad84ee23a20f2f17d/src/System.Console/src/System/Console.cs#L329